public/locations-list-map.js

```diff
@@ -1,5 +1,49 @@
 import toDataByLocation from './toDataByLocation.js';
 
+
+
+// TODO (patricknelson): The indentation in this file is not consistent and should be updated soon. Currently the project
+//  standard is 2-space indentation; please see .editorconfig and, if possible, configure your IDE to utilize this file.
+
+// TODO (patricknelson): All functions interacting with the map should not require the map as a parameter since this
```

Comment:
I don't agree with this TODO. The code thus far is written avoiding most globals. Let's keep that pattern until there's a strong reason to change.

Reply:
But not all yet they were merged into master... should this hold up this PR?